server/database/config.ts

```diff
@@ -0,0 +1,13 @@
+import mongoose from 'mongoose';
+import dotenv from 'dotenv';
+
+dotenv.config();
+const DB_URL = process.env.DB_URL || 'mongodb://localhost:27017/test';
+
+async function startDb() {
+  const db = await mongoose.connect(DB_URL);
+  console.log('Connected to DB');
+  return db;
+}
+
```

Comment:
Add try and catch and throw an error if there is any problem in connecting to the database. And I think it better to rename it to connectDb